l10n_ro_fiscal_validation/models/res_partner.py

```diff
@@ -21,8 +21,8 @@
     "Content-Type": "application/json;",
 }
 
-ANAF_BULK_URL = "https://webservicesp.anaf.ro/AsynchWebService/api/v6/ws/tva"
-ANAF_CORR = "https://webservicesp.anaf.ro/AsynchWebService/api/v6/ws/tva?id=%s"
+ANAF_BULK_URL = "https://webservicesp.anaf.ro/AsynchWebService/api/v7/ws/tva"
```

Comment:
Nu ar fi mai bine daca am pune versiunea in configurare ?

Reply:
Poate ca da...dar acum s-a schimbat tot response-ul, deci tot trebuie cod pe langa chiar daca e in config versiunea